### ISSUE TITLE
[BUG] Cannot reproduce results of NBEATS in Forecasting Hierarchical Example

### ISSUE THREAD

**Issue by jobs-git** [sktime/sktime#8262]
**Describe the bug**
Cannot reproduce results of NBEATS in Forecasting Hierarchical Example

**To Reproduce**
1. Run the section for NBEATS until preduction: https://github.com/sktime/sktime/blob/main/examples/01c_forecasting_hierarchical_global.ipynb
2. The y_pred will all be zero

```python
from sklearn.model_selection import train_test_split

from sktime.utils._testing.hierarchical import _make_hierarchical

data = _make_hierarchical(
    hierarchy_levels=(100, 1), max_timepoints=10, min_timepoints=10, n_columns=1
)
data = data.droplevel(1)
y_train, y_test = train_test_split(data, test_size=0.1, train_size=0.9, shuffle=False)

from sktime.forecasting.pytorchforecasting import PytorchForecastingNBeats

model = PytorchForecastingNBeats(
    trainer_params={
        "max_epochs": 5,  # for quick test
        "limit_train_batches": 50,  # for quick test
    },
    dataset_params={
        "max_encoder_length": 3,
    },
)

from sktime.forecasting.base import ForecastingHorizon

fh = ForecastingHorizon([1, 2, 3], is_relative=True)
model.fit(y=y_train, fh=fh)

y_pred = model.predict(y=y_test, fh=fh)
y_pred
```

**Expected behavior**
There would be predictions in y_pred, not zero

**Additional context**
n/a

**Versions**
<details>

</details>

**Comment by jobs-git** [comment]
Not a bug but a data set issue, solution: test another data set and make sure `loss` is acceptable
